Formula/arxiv_latex_cleaner

```diff
@@ -0,0 +1,37 @@
+class ArxivLatexCleaner < Formula
```

Comment:
This file needs to have a `.rb` extension. `brew` (or even GitHub) doesn't recognise it as a formula file otherwise.

Reply:
Yikes! Yes, it certainly does.